scavenger-frontend/src/components/snapshot/SnapshotTable.vue

```diff
@@ -85,6 +85,8 @@ import SnapshotForm from "./SnapshotForm.vue";
 import Momnet from "moment";
 import {useStore} from "../util/store";
 import {ElMessageBox, ElNotification} from "element-plus";
+import * as Xlsx from 'xlsx'
```

Comment:
Previously, the delimiter was tab. Is this the same in xsx?

Reply:
In xlsx, the default field separator delimiter is `,`